demo-project/src/demo_project/requirements.in

```
@@ -1,19 +1,17 @@
-black==21.5b1
+black==22.1.0
```

```suggestion
black~=22.0
```
for consistency with kedro.